swap_meet/vendor.py

```diff
@@ -0,0 +1,207 @@
+#_______________WAVE 1_______________________
+''' from item import Item
+from clothing import Clothing
+from decor import Decor
+from electronics import Electronics '''
+
+class Vendor:
+    def __init__(self, inventory = None): 
+        if inventory is None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+     
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        for thing in self.inventory:
+            if item == thing:
+                self.inventory.remove(thing)
+                return item
+            
+        return False         
+        
+#_______________WAVE 2_________________________
+    #every instance of the item class becomes an element in the inventory list
+    #returns list of class Item's objects inventory = [Item_a, Item_b, Item_c]
+    
+    def get_by_category(self, category):
+        new_list_with_items = []
+        
+        #iterate on the list for each category (state) and check if the item is in there 
+        for thing in self.inventory:
```

Comment:
Consider using the name `item` instead of `thing` to make it clear that the things in `self.inventory` are in fact instances of the `Item` class.